include/internal/libspdm_requester_lib.h

```diff
@@ -399,20 +422,22 @@ libspdm_return_t libspdm_send_spdm_request(libspdm_context_t *spdm_context,
  *
  * @param  spdm_context                  The SPDM context for the device.
  * @param  session_id                    Indicate if the response is a secured message.
- *                                     If session_id is NULL, it is a normal message.
- *                                     If session_id is NOT NULL, it is a secured message.
+ *                                       If session_id is NULL, it is a normal message.
+ *                                       If session_id is NOT NULL, it is a secured message.
+ * @param  support_large_response        Perform large response handling if necessary.
  * @param  response_size                 size in bytes of the response data buffer.
- * @param  response                     A pointer to a destination buffer to store the response.
- *                                     The caller is responsible for having
- *                                     either implicit or explicit ownership of the buffer.
- *                                      For normal message, response pointer still point to original transport_message.
- *                                      For secured message, response pointer will point to the scratch buffer in spdm_context.
+ * @param  response                      A pointer to a destination buffer to store the response.
+ *                                       The caller is responsible for having
+ *                                       either implicit or explicit ownership of the buffer.
+ *                                       For normal message, response pointer still point to original transport_message.
+ *                                       For secured message, response pointer will point to the scratch buffer in spdm_context.
  *
  * @retval RETURN_SUCCESS               The SPDM response is received successfully.
  * @retval RETURN_DEVICE_ERROR          A device error occurs when the SPDM response is received from the device.
  **/
 libspdm_return_t libspdm_receive_spdm_response(libspdm_context_t *spdm_context,
                                                const uint32_t *session_id,
+                                               bool support_large_response,
```

Comment:
One question for this: Do you see the absolute need to add this parameter?
What if we always support `support_large_response` ?

Reply:
Well, the only disadvantage I see with removing it is you could be unintentionally supporting CHUNK_GET on responses that theoretically should not need it. So a bug could be hidden.  An alternative though is remove the parameter and in the large response handler code after the large response is retrieved, assert that the response code is one of those responses we expect as valid for large response. That is cleaner.

Reply:
you can cache the request opcode in spdm_context, by using request opcode, you can already distinguish that without introducing new field.

Reply:
Good point. The request code is actually already there in the last_spdm_request member. The CHUNK_GET does not overwrite that member.

Reply:
I will push a change that removes the parameter, and will always try CHUNK_GET on an large response. Then it will assert if any of the responses are unexpected ones that shouldn't require a large response. If it passes CI.  I'll then create a new branch and pull request so as to not have any change history to the libspdm requester files that called this `libspdm_receive_spdm_response `function with the old `support_large_response` parameter.